src/v2/kubernetes_cluster/spec/api_server/types.rs

```diff
@@ -11,6 +11,7 @@ pub struct APIServerState {
     pub resources: StoredState,
     pub uid_counter: Uid,
     pub resource_version_counter: ResourceVersion,
+    pub used_names: Set<StringView>,
```

Comment:
@codyjrivera could you add a comment here to explain and remind ourselves in the future why we want this history variable?

Reply:
btw, don't you need to initialize `used_names` to empty in the initial state?

Reply:
Stale, I removed `used_names`.